tests/scripts/gen_release_notes.sh

```diff
@@ -22,7 +22,7 @@ pr_list=$(git log --pretty="%s" --merges --left-only "${FROM_BRANCH}"..."${TO_TA
 function release_notes() {
   for pr in $pr_list; do
   # get PR title
-  backport_pr=$(curl -s -u "${GITHUB_USER}":"${GITHUB_TOKEN}" "https://api.github.com/repos/rook/rook/pulls/${pr}" | jq '.title')
+  backport_pr=$(curl -s -u "${GITHUB_USER}":"${GITHUB_TOKEN}" "https://api.github.com/repos/koor-tech/koor/pulls/${pr}" | jq '.title')
```

Comment:
Maybe we can use `.env` for this in the future

Reply:
@galexrt  how about we do it in coming PR, since this will help with 1.11 koor release

Reply:
Sure sounds good